CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ## Bug Fixes
 
+- PR #5306 Fix categorical scaler insertion
```

Comment:
```suggestion
- PR #5306 Fix categorical scalar insertion
```

Reply:
Please spell check docs and changelog entries.

Reply:
Sorry about this, have added a spell checker to my ide to prevent this from happening again.